go-controller/pkg/ovn/pods.go

```diff
@@ -28,10 +28,14 @@ func (oc *Controller) syncPods(pods []interface{}) {
 			klog.Errorf("Spurious object in syncPods: %v", podInterface)
 			continue
 		}
-		_, err := util.UnmarshalPodAnnotation(pod.Annotations)
+		annotations, err := util.UnmarshalPodAnnotation(pod.Annotations)
 		if podScheduled(pod) && podWantsNetwork(pod) && err == nil {
 			logicalPort := podLogicalPortName(pod)
 			expectedLogicalPorts[logicalPort] = true
+			podIP := annotations.IPs[0].IP
+			if err = oc.lsCache.AllocateIP(pod.Spec.NodeName, &podIP); err != nil {
+				klog.Errorf("Couldn't sync allocated IP: %s with IPAM cache for pod: %s on node: %s", podIP, logicalPort, pod.Spec.NodeName)
+			}
```

Comment:
L37: is too long can you please move the parameters to the next line

Reply:
done.